.github/workflows/inactive_issues.yml

```
@@ -16,6 +16,8 @@ jobs:
           days-before-issue-close: 14
           stale-issue-label: 'needs-more-info'
           close-issue-message: 'Задача закрыта из-за отсутствия активности в течение последних 14 дней. Если это произошло по ошибке или проблема все ещё актуальна, откройте задачу повторно.'
-          days-before-pr-stale: -1
-          days-before-pr-close: -1
+          days-before-pr-stale: 7
+          days-before-pr-close: 7
+          stale-pr-label: 'needs-work'
```

Мб сделать `no-pr-activity`, чтобы было более понятней

```suggestion
          stale-pr-label: 'no-pr-activity'
```

---

`needs-work` тоже норм

но возможно стоит добавить префиксы

Типа
`issue-needs-more-info`
`pr-needs-work`